pubsub/google/cloud/pubsub/topic.py

```diff
@@ -252,7 +250,7 @@ def publish(self, message, client=None, **attrs):
         api = client.publisher_api
 
         self._timestamp_message(attrs)
-        message_b = base64.b64encode(message).decode('ascii')
+        message_b = message.decode('ascii')
```

Comment:
It seems `.decode('ascii')` is only in there for Python 3, to make sure `message_b` to be (`str` (py3) / `unicode` (py2)). It's unclear why this is needed?